extension/resources/templates/report_class.liquid

```diff
@@ -0,0 +1,86 @@
+<head>

```

Comment:
Just some opinion that the liquid template is not syntactically condensed. Here's an example of Jade:
```jade
doctype html
html(lang="en")
  head
    title= pageTitle
    script(type='text/javascript').
      if (foo) bar(1 + 5)
  body
    h1 Jade - node template engine
    #container.col
      if youAreUsingJade
        p You are amazing
      else
        p Get on it!
      p.
        Jade is a terse and simple templating language with a
        strong focus on performance and powerful features.
```

Not sure why you chose to use liquid-node which only has 698 downloads last week while jade has 535,984...

Reply:
I haven't heard about Jade before. After some investigation, Jade's API doesn't support async and it cannot load templates from a directory, however it supports more complex syntax than liquid. since jade is more popular and may be better maintained, switch to jade is ok.